### ISSUE TITLE
Softmax layers on output of models

### ISSUE THREAD

**Issue by hzavadil98** [SFI-Visual-Intelligence/Collaborative-Coding-Exam#37]
Hi, in the main.py in the training loop we are currently using `nn.CrossEntropyLoss()`, it should take as input the raw logits (not softmaxed probabilities) and targets - [https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html](CrossEntropyLoss) if I'm not mistaken.
